# HG changeset patch
# User Daiki Ueno <dueno@redhat.com>
# Date 1487602422 -3600
#      Mon Feb 20 15:53:42 2017 +0100
# Branch wip/dueno/tstclnt-optstate
# Node ID ec284d402a5a691e2694fe27d8ab2e95d525f5ab
# Parent  ec6b5abc4187459458779d1e90bc8500a011eb3a
tstclnt: use correct option spec for -W

diff --git a/cmd/tstclnt/tstclnt.c b/cmd/tstclnt/tstclnt.c
--- a/cmd/tstclnt/tstclnt.c
+++ b/cmd/tstclnt/tstclnt.c
@@ -1509,7 +1509,7 @@ main(int argc, char **argv)
     /* XXX: 'B' was used in the past but removed in 3.28,
      *      please leave some time before resuing it. */
     optstate = PL_CreateOptState(argc, argv,
-                                 "46A:CDFGHI:KL:M:OR:STUV:WYZa:bc:d:fgh:m:n:op:qr:st:uvw:z");
+                                 "46A:CDFGHI:KL:M:OR:STUV:W:YZa:bc:d:fgh:m:n:op:qr:st:uvw:z");
     while ((optstatus = PL_GetNextOpt(optstate)) == PL_OPT_OK) {
         switch (optstate->option) {
             case '?':
